charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -101,35 +100,21 @@ allowedMetrics:
   - etcd_server_proposals_pending
   - grpc_server_handled_total
   - grpc_server_started_total
-  - grpc_server_handling_seconds_count
-  - grpc_server_handling_seconds_sum
-  - grpc_server_handling_seconds_bucket
   - process_max_fds
   - process_open_fds
   - process_resident_memory_bytes
   kubeETCD3BR:
-  - etcdbr_defragmentation_duration_seconds_bucket
```

Comment:
@shreyas-s-rao is working on building grafana dashboard around these metrics from etcd-backup-restore. So, please don't remove them. These are required from audit perspective as well. He will get in touch with you regarding same soon.

Reply:
etcdbr metrics have been readded.